trustfall_core/test_data/tests/valid_queries/fold_with_no_outputs_transform_and_filter_less_than.graphql.ron

```diff
@@ -0,0 +1,18 @@
+TestGraphQLQuery (
+    schema_name: "numbers",
+    query: r#"
+{
+    Number(min: 30, max: 30) {
+        ... on Composite {
+            value @output
+
+            primeFactor @fold @transform(op: "count") @filter(op: "<", value: ["$min_primes"]) {
+                __typename
+            }
+        }
+    }
+}"#,
+    arguments: {
+        "min_primes": Uint64(2)
+    },
```

Comment:
Can you tweak this query so it returns at least one result? That way we won't have to stare too hard at the trace if we have a bug that causes all results to disappear.

Perhaps just make the entrypoint have `max: 32` which should return just 32 (since 31 is prime).